CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+Contributing to Truffle
+=======================
+
+_Thanks for taking the time to help out and improve Truffle!_
+
+The following is a set of guidelines for Truffle contributions. This document
+is intended to provide guidelines, and may change over time. Feel free to
```

Comment:
These two sentences feel redundant: 

> The following is a set of guidelines for Truffle contributions.
> This document is intended to provide guidelines, and may change over time.

Perhaps merge them: 

> The following is a set of guidelines for Truffle contributions, and may change over time.

Side note: I think it's awesome you request PRs for this document specifically.